dali/kernels/slice/slice_cpu.h

```diff
@@ -287,10 +375,28 @@ class SliceCPU {
     return req;
   }
 
+  /**
+   * @brief Run the kernel with an execution engine.
+   *
+   *        For execution engines other than
+   *        SequentialExecutionEngine, the algorithm will try to split the slice into
+   *        similar sized blocks until we either reach a minimum of ``req_nblocks`` or
+   *        the block volume is smaller than the minimum practical size, ``min_blk_sz``.
+   * @param context Kernel context
+   * @param out Output tensor view
+   * @param in Input tensor view
+   * @param slice_args Slice arguments
+   * @param min_blk_sz Minimum practical block size
+   * @param req_nblocks Requested number of blocks. By default the requested number of blocks
+   *                    is calculated as ``num_threads * 8``
+   */
+  template <typename ExecutionEngine = SequentialExecutionEngine>
```

Comment:
```suggestion
  template <typename ExecutionEngine>
```
There's no need for the default type if the argument needs to be supplied anyway.